srcpkgs/gala/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gala'
+pkgname=gala
+version=3.2.0
+revision=1
+build_style=meson
+hostmakedepends="cmake pkg-config gettext glib-devel"
+makedepends="gnome-desktop-devel vala-devel bamf-devel plank-devel granite-devel gnome-settings-daemon-devel mutter-devel libcanberra-devel"
+short_desc="Gala Window Manager"
+maintainer="linarcx <linarcx@riseup.net>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/elementary/gala"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=137ceb10f9649defd30c27606fb63eb82c3622eb8aa45339747c135b03d723a8
+
+gala-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/*.so
+		vmove /usr/lib/libgala.so.0.0.0
```

Comment:
this is definitely wrong. shared libraries should be part of the main pkg, not development.

Reply:
What about [this](https://github.com/void-linux/void-packages/blob/12680196c9980219f6285ae2fd3ff02958ed82bd/srcpkgs/budgie-desktop/template#L31)?

Reply:
what about [this](https://github.com/void-linux/void-packages/blob/12680196c9980219f6285ae2fd3ff02958ed82bd/srcpkgs/budgie-desktop/template#L31)?

Reply:
yes, this is correct.

Reply:
Ok, I'll remove [this line](https://github.com/void-linux/void-packages/pull/20538/files/ba11dc89f217344fa06549e549b75b0fa449e8ad) too.

Reply:
Interesting!
I've got this warning during build of gala:
> => WARNING: usr/lib/*.so should be in -devel package

Reply:
Also this one:
> => WARNING: gala-3.2.0_1: libgala.so.0 not found in common/shlibs!

Reply:
Ok. It [failed](https://gist.github.com/LinArcX/32a967d872ec8f348dec92cc18cb9a6f#file-failed-L27) when i depend wingpanel to gala:

Reply:
Done!